api/hearings/hearing.mock.ts

```diff
@@ -18,7 +18,7 @@ export const init = () => {
 
   const cancelHearingRequest = /https:\/\/hearings.aat.service.core-compute-aat.internal\/hearing\/[\w]*/;
 
-  const getHearingsActualsUrl = /https:\/\/hearings.aat.service.core-compute-aat.internal\/actuals\/[\w]*/;
+  const hearingsActualsUrl = /https:\/\/hearings.aat.service.core-compute-aat.internal\/hearingActuals\/[\w]*/;
```

Comment:
it also needs to amend 
getHearingActuals, to match this url pattern.
it's now:
`export async function getHearingActuals(req: EnhancedRequest, res: Response, next: NextFunction): Promise<void> {
  const hearingId = req.params.hearingId;
  try {
    const { status, data }: { status: number, data: HearingActualsMainModel } =
      await handleGet(`${hearingsUrl}/actuals/${hearingId}`, req);
    res.status(status).send(data);
  } catch (error) {
    next(error);
  }
}`